src/main/scala/viper/gobra/frontend/ParseTreeTranslator.scala

```diff
@@ -2071,7 +2086,24 @@ class ParseTreeTranslator(pom: PositionManager, source: Source, specOnly : Boole
     * @return the visitor result
     */
   override def visitImportDecl(ctx: GobraParser.ImportDeclContext): Vector[PImport] = {
-    visitListNode[PImport](ctx.importSpec())
+    val importsVector: Vector[PImport] = visitListNode[PImport](ctx.importSpec())
+    val importPres: Vector[PExpression] = visitListNode[PExpression](ctx.importPre())
+    // if there is only a single importSpec and the importDecl has specification,
+    // then update the specification of the importSpec with the one from the importDecl
```

Comment:
I changed the code in order to produce more informative error messages when parsing imports. There is currently a bug which causes the error messages produced by `fail` in this method to cause an ANTLR4Exception (because no position information for `ctx` is found, where `ctx` is the parameter to `visitImportDecl`. This causes the parsing error to be reported together with an ugly error message. I did not manage to understand the cause of this yet though.